src/index.html

```diff
@@ -9,10 +9,56 @@
     <title>Stars</title>
     <link
```

Comment:
Keep your attributes correctly formatted. If the HTML-element has long attribute values or a number of attributes is more than 2, start each one, including the first, on a new line with 2-space indentation relative to the tag. The tag’s closing bracket should be on the same level as the opening one.